### PR TITLE
Updated dump of SIM geometry

### PR DESCRIPTION
**Pull request by civanch**
Dump of CMS geometry into gdml format has been updated: regions and cuts are dumped, name extensions are not applied.

Added extra option to dump G4Regions and cuts into txt file.

Should not affect simulation itself.

### CODE REVIEW

**Comment by civanch** [comment]
+1

**Comment by civanch** [comment]
please test

File: SimG4Core/Application/interface/RunManagerMT.h [hunk 1]
```diff
@@ -112,7 +111,7 @@ class RunManagerMT
   std::unique_ptr<PhysicsList> m_physicsList;
   bool m_managerInitialized;
   bool m_runTerminated;
-  const bool m_pUseMagneticField;
+  bool m_pUseMagneticField;
```
Comment (makortel) [inline SimG4Core/Application/interface/RunManagerMT.h]: Out of curiosity, why this can't be const?

**Comment by civanch** [comment]
@makortel , it is initialisation parameter - to have only one of them const looks strange - should all or no one. In general doe not matter because is used once.

**Comment by makortel** [comment]
Sounds reasonable.

**Comment by davidlange6** [comment]
+1

**Pull request merged**